database_admin/schema/create_schema.sql

```diff
@@ -938,7 +938,7 @@ SELECT DISTINCT ON (p.name_id) p.name_id, p.id as package_id, sum.value as summa
 FROM package p
          INNER JOIN strings sum on p.summary_hash = sum.id
          LEFT JOIN advisory_metadata am on p.advisory_id = am.id
-ORDER BY p.name_id, am.public_date DESC NULLS LAST;
+ORDER BY p.name_id, am.public_date DESC NULLS LAST, sum.value DESC NULLS LAST;
```

Comment:
@semtexzv it doesn't work. Try this in QA:
~~~SQL
select * from (
SELECT DISTINCT ON (p.name_id) p.name_id, p.id as package_id, sum.value as summary
FROM package p
         INNER JOIN strings sum on p.summary_hash = sum.id
         LEFT JOIN advisory_metadata am on p.advisory_id = am.id
ORDER BY p.name_id, am.public_date DESC NULLS LAST, sum.value DESC NULLS LAST) t
where summary = '';
~~~
... it returns many rows.

Reply:
It returns 1700 rows vs 12000 returned by the query before. It seems we have some content, which has no summaries. From my exploration it's just `.src` only packages

Reply:
You're right. I've also tested that. All packages without summary are `.src`.